packages/common/src/contracts/usdt.ts

```diff
@@ -0,0 +1,192 @@
+export const usdtABI = [
+  {
+    type: 'event',
+    name: 'Approval',
+    inputs: [
+      {
```

Comment:
do we really need the full abi? can we use the compact one for just the transfer method?

Reply:
done

Reply:
i guess we could only use the compact abi for erc20, right?